ReactNativeClient/lib/services/KeymapService.ts

```diff
@@ -0,0 +1,202 @@
+const BaseService = require('lib/services/BaseService');
+const { Logger } = require('lib/logger.js');
+const { shim } = require('lib/shim.js');
+
+class KeymapService extends BaseService {
+	private keymap: { [command: string]: KeymapItem };
+
+	static modifiers = /^(CommandOrControl|Command|Control|Ctrl|Cmd|CmdOrCtrl|Alt|Option|AltGr|Shift|Super)$/;
+	static keyCodes = /^([0-9A-Z)!@#$%^&*(:+<_>?~{|}";=,\-./`[\\\]']|F1*[1-9]|F10|F2[0-4]|Plus|Space|Tab|Backspace|Delete|Insert|Return|Enter|Up|Down|Left|Right|Home|End|PageUp|PageDown|Escape|Esc|VolumeUp|VolumeDown|VolumeMute|MediaNextTrack|MediaPreviousTrack|MediaStop|MediaPlayPause|PrintScreen)$/;
+
+	static defaultKeymap: KeymapItem[] = [
+		{ accelerator: 'CommandOrControl+N', command: 'newNoteItem', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+T', command: 'newTodoItem', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+S', command: 'synchronise', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+P', command: 'printItem', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+H', command: 'hideJoplin', platforms: ['darwin'] },
+		{ accelerator: 'CommandOrControl+Q', command: 'quit', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+,', command: 'preferences', platforms: ['darwin'] },
+		{ accelerator: 'Command+W', command: 'closeWindow', platforms: ['darwin'] },
+		{ accelerator: 'CommandOrControl+Alt+I', command: 'insertTemplate', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+C', command: 'copy', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+X', command: 'cut', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+V', command: 'paste', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+A', command: 'selectAll', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+B', command: 'textBold', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+I', command: 'textItalic', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+K', command: 'textLink', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+`', command: 'textCode', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+T', command: 'insertDateTime', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: shim.isMac() ? 'Shift+Command+F' : 'F6', command: 'focusSearch', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
```

Comment:
Why is there a dynamic check for the platform when you already have a platform property. There should be two shortcuts:

```
{ accelerator: 'Shift+Command+F', command: 'focusSearch', platforms: ['darwin'] },
{ accelerator: 'F6', command: 'focusSearch', platforms: ['win32', 'linux', 'freebsd'] },
```